src/ML-QUIP/pair_quip.cpp

```diff
@@ -314,8 +318,16 @@ void PairQUIP::init_style()
 /* ----------------------------------------------------------------------
    init for one type pair i,j and corresponding j,i
 ------------------------------------------------------------------------- */
+double PairQUIP::init_one(int i, int j)
```

Comment:
The init_one function can be restored to its original since the arguments are ignored.